helm-charts/strimzi-kafka-operator-helm3/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+appVersion: "0.1.0"
+description: "Strimzi: Apache Kafka running on Kubernetes"
+name: strimzi-kafka-operator-helm3
```

Comment:
You could restructure the folder layout to avoid needing to rename the chart. That might be nicer than building something with "helm3" in the name.

e.g. `helm-charts/strimzi-kafka-operator/Chart.yaml` could be the existing one, and you could add
`helm-3-charts/strimzi-kafka-operator/Chart.yaml` for the new one, which can still be called strimzi-kafka-operator

That way each folder could have it's own Makefile, one built and linted with Helm 2, the other built and linted with Helm 3.

Or, if you want to keep them all in a single folder with a single Makefile, you could do something like:
`helm-charts/helm-2/strimzi-kafka-operator/Chart.yaml` for the existing one, and
`helm-charts/helm-3/strimzi-kafka-operator/Chart.yaml` for the new one